pkg/testgridanalysis/testidentification/test_identification.go

```diff
@@ -24,7 +24,7 @@ var customJobSetupContainers = sets.NewString(
 	"e2e-metal-ipi-ovn-ipv6-baremetalds-devscripts-setup",
 	"e2e-metal-ipi-upgrade-baremetalds-packet-setup",
 	"e2e-metal-ipi-virtualmedia-baremetalds-devscripts-setup",
-	"e2e-ovirt-ipi-install-ovirt-setup container test",
+	"e2e-ovirt-ipi-install-install container test",
```

Comment:
do the 4.7 and older jobs still use the other name? if so please leave it so the other release pages do not get an error.

Reply:
No they changed as well...
Not sure why it works:
See example on:
4.5 https://prow.ci.openshift.org/view/gcs/origin-ci-test/logs/periodic-ci-openshift-release-master-ocp-4.5-e2e-ovirt/1360332918110556160
4.7 https://prow.ci.openshift.org/view/gcs/origin-ci-test/logs/periodic-ci-openshift-release-master-ocp-4.7-e2e-ovirt/1361556060103512064
4.8 https://prow.ci.openshift.org/view/gcs/origin-ci-test/logs/periodic-ci-openshift-release-master-ocp-4.8-e2e-ovirt/1360472590920978432

Reply:
cool, thanks